bin.src/validateCfht.py

```diff
@@ -62,5 +58,5 @@ def defaultData(repo):
         print("Could not find repo %r" % (repo,))
         sys.exit(1)
 
-    visitDataIds, refDataIds, good_mag_limit, medianRef, matchRef = defaultData(repo)
-    checkAstrometry.run(repo, visitDataIds, refDataIds, good_mag_limit, medianRef, matchRef)
+    args = defaultData(repo)
```

Comment:
I realise it's only being use to copy from one line to the next, but `args` isn't a super descriptive name! It'd be better just to do away with it, I think
```
checkAstrometryPhotometry.run(repo, *defaultData(repo))
```
or similar.